console/src/tasks.rs

```diff
@@ -373,14 +373,18 @@ impl From<proto::tasks::Stats> for Stats {
             total,
             idle,
             busy,
-            last_poll_started: pb.last_poll_started.map(Into::into),
-            last_poll_ended: pb.last_poll_ended.map(Into::into),
+            last_poll_started: pb.last_poll_started.map(|v| v.try_into().unwrap()),
+            last_poll_ended: pb.last_poll_ended.map(|v| v.try_into().unwrap()),
             polls: pb.polls,
-            created_at: pb.created_at.expect("task span was never created").into(),
+            created_at: pb
+                .created_at
+                .expect("task span was never created")
+                .try_into()
+                .unwrap(),
             wakes: pb.wakes,
             waker_clones: pb.waker_clones,
             waker_drops: pb.waker_drops,
-            last_wake: pb.last_wake.map(Into::into),
+            last_wake: pb.last_wake.map(|v| v.try_into().unwrap()),
```

Comment:
hmm, not totally sure if panicking is the right approach when the timestamp conversion from protobuf fails...but i'm guessing it's what would have happened anyway in the previous version of `prost`?

we can fix this later.